src/DrugstoreApp/Backend/DrugstoreAPI/DrugstoreAPI/Controllers/HospitalController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using DrugstoreAPI.Filters;
+using RestSharp;
+using DrugstoreAPI.Repository;
+using Integration.Repository.Sql;
+using Drugstore.Models;
+using Service;
+using DrugstoreAPI.Models;
+using DrugstoreAPI.Dtos;
+
+namespace DrugstoreAPI.Controllers
+{
+    [Route("api/[controller]")] // /api/DrugstoreFeedbackWithAPI
+    [ApiController]
+
+    public class HospitalController : ControllerBase
+    {
+        private readonly MyDbContext dbContext;
+        public HospitalService hospitalService = new HospitalService();
+
+
+        public HospitalController(MyDbContext db) //Ovo mora da stoji, ne znam zasto!!!
+        {
+            this.dbContext = db;
+        }
+
+        [HttpPost]
+        public IActionResult Post(HospitalDto hospital)
+        {
+            hospitalService = new HospitalService(dbContext);
+            int id = hospitalService.GetMaxId();
```

Comment:
ne ikrementovati ručno id, postoji tip polja unutar baze koji to radi sam

Reply:
MAX ID CE BITI SREDJEN !!!! Sto pre